schema/2.0.0/corpora_schema.md

```diff
@@ -256,7 +260,7 @@ Curators MUST annotate the following columns in the `obs` dataframe:
     </tr>
     <tr>
       <th>Value</th>
-        <td>categorical with <code>str</code> categories. If unavailable, this MUST be <code>"unknown"</code>.<br><br>If <code>organism_ontolology_term_id</code> is <code>"NCBITaxon:9606"</code> for <i>Homo sapiens</i>, this MUST be the most<br>accurate HsapDv term with the following STRONGLY RECOMMENDED:
+        <td>categorical with <code>str</code> categories. If unavailable, this MUST be <code>"unknown"</code>.<br><br>If <code>organism_ontolology_term_id</code> <b>is not</b> <code>"NCBITaxon:9606"</code> (<i>Homo sapiens</i>) <br> <b>nor</b> <code>"NCBITaxon:10090"</code> (<i>Mus musculus</i>), this MUST be the most accurate child of <code>EFO:0000399</code>. <br><br>If <code>organism_ontolology_term_id</code> is <code>"NCBITaxon:9606"</code> for <i>Homo sapiens</i>, this MUST be the most<br>accurate HsapDv term with the following STRONGLY RECOMMENDED:
```

Comment:
It would move this below the other requirements for Homo sapiens and Mus musculus. 

Otherwise, this MUST be the most accurate child of EFO:0000399.